rootfs/api/models/release.py

```diff
@@ -95,7 +95,7 @@ def publish(self, source_version='latest'):
             source_image.startswith(settings.REGISTRY_HOST) or
             source_image.startswith(settings.REGISTRY_URL)
         ):
-            logger.debug('{} already exists in the target registry. Using this image for release {} of app {}'.format(source_image, self.version, self.app))  # noqa
+            log_event(self.app, '{} already exists in the target registry. Using this image for release {} of app {}'.format(source_image, self.version, self.app))  # noqa
```

Comment:
Not directly related beyond I realised I was relaying information in the wrong context and made this match what I did in the delete exception